pkg/server/upload.go

```diff
@@ -96,26 +86,39 @@ func (s *agentServer) UploadManifest(ctx context.Context,
 		}
 	}()
 
+	// collect all file status updates in a single buffered channel to serialize writes
+	statusUpdates := make(chan models.UploadStatusUpdateMessage, 100)
+	go s.startStatusUpdateBatchWriter(statusUpdates)
+
 	tickerDone := make(chan bool)
 	ticker := time.NewTicker(10 * time.Second)
 
 	// Ticker to get status updates from the server periodically
-	syncTickerDelay := 15 * time.Minute // time to continue syncing files after upload complete
```

Comment:
Does this mean we are now stopping to sync the manifest after upload is complete? Originally we continue to sync after upload is complete to allow server to process and move files. (I don't think it matters either way though)